gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfig.java

```diff
@@ -230,7 +231,15 @@ static void action(Params params) {
     // https://mail.openjdk.org/pipermail/hotspot-compiler-dev/2023-September/068447.html
     // AVX-512 is largely unreliable, so we're going to opt out entirely for the time
     // being.
-    private static List<String> getAvxOptions(Params _params) {
+    // Update for the October 17, 2023 CPU hotfixes: The above issue has been resolved,
+    // we will allow avx-512 instructions initially only for jdk21+ to build confidence,
+    // and align rollout with the new jdk. Assuming this goes well, we may allow jdk17
+    // to use avx-512 instructions in a future sls-packaging release.
+    private static List<String> getAvxOptions(Params params) {
+        JavaVersion javaVersion = params.getJavaVersion().get();
+        if (javaVersion.compareTo(JavaVersion.toVersion("21")) >= 0) {
+            return Collections.emptyList();
+        }
         return disableAvx512;
```

Comment:
I'm ok with this as for any JDK 21+ service we can disable if needed via `-XX:UseAVX=2` JVM arg